style/index.qmd

```diff
@@ -1,5 +1,25 @@
 ---
-title: "Style"
+title: "Style guidelines"
 ---
 
 {{< include /includes/_wip.qmd >}}
+
+## Purpose
+
+These documents contained within the style section are meant to provide
+guidance on how to write, format, and structure documents within the
+Seedcase project. They are meant to be a guide and a reference for all
```

Comment:
```suggestion
Seedcase Project. They are meant to be a guide and a reference for all
```
I personally don't mind either way, but it's in the [terminology section](https://team.seedcase-project.org/how-we-work/terminology), so I guess I'll follow it